### DISCUSSION TITLE
Installation details

### DISCUSSION THREAD

**Discussion by harrycolorado** [AllenInstitute/ZeroMQ-XOP#39] [Q&A] [answered]
Hi,
I just tried to install this package on Windows and, after cloning the repository to my computer, got stuck at the first item "Install the vcredist packages in "output/win". I was not able to find anything labeled "vcredist" in that folder. I suspect I'm missing something trivial and would appreciate some help.

**Comment by t-b** [comment, marked as answer]
Thanks for the feedback.

The installation instructions have bitrotted and are not correct anymore.

I'm attaching a zip archive with the XOP compiled for Windows/Mac.[ZeroMQ-XOP-version-231-g6061e65.zip](https://github.com/AllenInstitute/ZeroMQ-XOP/files/9114545/ZeroMQ-XOP-version-231-g6061e65.zip)

Extract that zip file to a folder. And now create the links as described in the installation instructions but ignoring the vcredist step.

**Reply by harrycolorado** [reply]
Thank you, that worked!
Forgive me for making baby steps here, but I was trying to continue by installing ZeroMQ, but I couldn't find a Windows installer. Am I missing something else again?

**Reply by t-b** [reply]
There is no clickable installer. You have to extract the zip file given above and create links, or copy the files, to the respective igor pro folders as described in https://github.com/AllenInstitute/ZeroMQ-XOP#installation.

**Reply by t-b** [reply]
I've updated the installation instructions.
